contentcuration/contentcuration/forms.py

```diff
@@ -288,13 +287,13 @@ def __init__(self, *args, **kwargs):
         class_attrs = {'class': 'form-control setting_change'}
 
         language_options = [(l['id'], _(l['readable_name'])) for l in Language.objects.values('id', 'readable_name').order_by('readable_name')]
-        language_options.insert(0, (None, _("Select a language")))  # Add default option if no language is selected
+        language_options.insert(0, (None, _("No language selected")))  # Add default option if no language is selected
```

Comment:
Updated here to avoid having to manually go in and disable this- just allow it to be selectable with the appropriate text